uint8 IDLE=0
uint8 SAMPLE_AUTOMATIC = 1
uint8 SAMPLE_ROI = 2
uint8 SAMPLE_CONTOURS = 3
uint8 SAMPLE_BORDER = 4
uint8 mode
---
bool success
